pom.xml

```diff
@@ -19,6 +19,11 @@
             <comments>All source code is under the MIT license.</comments>
         </license>
     </licenses>
+    
+    <properties>
+      <jenkins.version>1.625</jenkins.version>
+      <java.level>6</java.level>
+    </properties>
```

Comment:
Use the profiles at the end of the parent POM as an orientation of the java levels to use (they are LTS boundaries, not exact version in which it change, but serve as an quick guide).